.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,5 +2,16 @@
 
 *Description of changes:*
 
+----
+*Testing*  
+- [ ] Did you run our unit tests? `make test`
+- [ ] Did you run our integration tests? `make integ-test`
+- [ ] Did you add new unit test(s)?
+- [ ] Did you add new integration test(s)?
```

Comment:
How about we also add a section for listing manual test cases?

Reply:
Applicable cases could change depending on what functionality as added, but minimally we could have section like, "List manual checks done and their output (include default case, embellished case(s), new/old functionality, etc.)".

Reply:
Added:

- [ ] Listed manual checks and their outputs in the comments ([example](https://github.com/aws/amazon-ecs-cli/pull/750#issuecomment-472623042))